flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/api/TableEnvironmentTest.scala

```diff
@@ -106,10 +107,23 @@ class TableEnvironmentTest {
   @Test
   def testStreamTableEnvironmentExecutionExplain(): Unit = {
     val execEnv = StreamExecutionEnvironment.getExecutionEnvironment
-    execEnv.setParallelism(1)
+    execEnv.setParallelism(4)
     val settings = EnvironmentSettings.newInstance().inStreamingMode().build()
-    val tEnv = StreamTableEnvironment.create(execEnv, settings)
+    var tEnv = StreamTableEnvironment.create(execEnv, settings)
+
+    testStreamTableEnvironmentExecutionExplain(tEnv)
+
+    tEnv = StreamTableEnvironment.create(
+      StreamExecutionEnvironment.getExecutionEnvironment,
+      settings)
+    val conf = new Configuration()
+    conf.setInteger("parallelism.default", 4)
+    tEnv.getConfig.addConfiguration(conf)
+
+    testStreamTableEnvironmentExecutionExplain(tEnv)
```

Comment:
I would like to separate them into different methods, then the failed message will be more clear to help find which is failed. 

For example, we can have `testExplainExecutionPlanWithEnvParallelism` and `testExplainExecutionPlanWithConfParallelism`.